Updating to LIS 4.2.2-1

PedroPerezMSFT opened the pull request:
Instructions for CentOS/RedHat pointed to LIS 4.2.1-1 and we're on 4.2.2-1 (and actually 4.2.1 doesn't seem to work with the current CentOS 7.3 in the marketplace)

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PedroPerezMSFT committed:
Updating to LIS 4.2.2-1

Instructions for CentOS/RedHat pointed to LIS 4.2.1-1 and we're on 4.2.2-1 (and actually 4.2.1 doesn't seem to work with the current CentOS 7.3 in the marketplace)
